texts/ContainerDictionary.md

```diff
@@ -5296,6 +5337,40 @@ Possible values: APPROVED, MANUAL_REVIEW.<br />
 	<div style="padding-bottom: 10px">Optional. Options which only apply to Klarna.</div>
 	<i class="small text-muted">
 			</i>
+</td>
+							</tr>
+					</tbody>
+				</table>
+			<h2>Container "Payment_Models_RiskDetails"</h2>

```

Comment:
Da ist noch mehr als nur ApplePay mitgekommen

Reply:
seems to be a reoder problem caused by the new container